nipype/interfaces/ants/segmentation.py

```diff
@@ -594,6 +594,105 @@ def _list_outputs(self):
+                                        'averaged resulting in a probability image.'),
+                                  copyfile=False, mandatory=True)
+    out_prefix = traits.Str('highres001_', argstr='-o %s', usedefault=True,
+                            desc=('Prefix that is prepended to all output'
+                                  ' files (default = highress001_)'))
+
+    extraction_registration_mask = File(exists=True, argstr='-f %s',
+                                        desc=('Mask (defined in the template space) used during'
+                                              ' registration for brain extraction.'
+                                              'To limit the metric computation to a specific region.'))
+    image_suffix = traits.Str('nii.gz', desc=('any of standard ITK formats,'
+                                              ' nii.gz is default'),
+                              argstr='-s %s', usedefault=True)
+    use_random_seeding = traits.Enum(0, 1, argstr='-u %d',
+                                     desc=('Use random number generated from system clock in Atropos'
+                                           '(default = 1)'))
+    keep_temporary_files = traits.Int(argstr='-k %d',
+                                      desc='Keep brain extraction/segmentation warps, etc (default = 0).')
+    use_floatingpoint_precision = traits.Enum(0, 1, argstr='-q %d',
+                                              desc=('Use floating point precision '
+                                                    'in registrations (default = 0)'))
+    debug = traits.Bool(argstr='-z 1',
+                        desc=('If > 0, runs a faster version of the script.'
+                              'Only for testing. Implies -u 0.'
+                              'Requires single thread computation for complete reproducibility.'))
+
+
+class antsBrainExtractionOutputSpec(TraitedSpec):
+    BrainExtractionMask = File(exists=True, desc='brain extraction mask')
+    BrainExtractionBrain = File(exists=True, desc='brain extraction image')
+
+class antsBrainExtraction(ANTSCommand):
+    """
+    Examples
+    --------
+    >>> from nipype.interfaces.ants.segmentation import antsBrainExtraction
+    >>> brainextraction = antsBrainExtraction()
+    >>> brainextraction.inputs.dimension = 3
+    >>> brainextraction.inputs.anatomical_image ='T1.nii.gz'
+    >>> brainextraction.inputs.brain_template = 'study_template.nii.gz'
+    >>> brainextraction.inputs.brain_probability_mask ='ProbabilityMaskOfStudyTemplate.nii.gz'
+    >>> brainextraction.cmdline
+    'antsBrainExtraction.sh -a T1.nii.gz -m ProbabilityMaskOfStudyTemplate.nii.gz -e study_template.nii.gz -d 3 -s nii.gz -o highres001_'
+    """
+    input_spec = antsBrainExtractionInputSpec
+    output_spec = antsBrainExtractionOutputSpec
+    _cmd = 'antsBrainExtraction.sh'
+
+    def _format_arg(self, opt, spec, val):
```

Comment:
There shouldn't be any need to overload this function for these inputs as they are pretty straight forward. I would recommend removing this for code clarity.